FTC SDK/FtcRobotController/src/main/java/com/qualcomm/ftcrobotcontroller/opmodes/OmniTeleOp.java

```diff
@@ -0,0 +1,79 @@
+package com.qualcomm.ftcrobotcontroller.opmodes;
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.util.Range;
+
+/**
+ * Created by Will Marchesi on 10/16/2015.
+ */
+public class OmniTeleOp extends OpMode{
+
+    DcMotor motorFrontLeft;
+    DcMotor motorFrontRight;
+    DcMotor motorBackLeft;
+    DcMotor motorBackRight;
+
+    public OmniTeleOp() {
+
+    }
+
+    @Override
+    public void init() {
+
+        motorFrontLeft = hardwareMap.dcMotor.get("motorFrontLeft");
+        motorFrontRight = hardwareMap.dcMotor.get("motorFrontRight");
+        motorBackLeft = hardwareMap.dcMotor.get("motorBackLeft");
+        motorBackRight = hardwareMap.dcMotor.get("motorBackRight");
+
+        telemetry.addData("Text", "*** Robot Data***");
+
+    }
+
+    @Override
+    public void loop() {
+
+        float driverLeftStick = -gamepad1.left_stick_y;
+        float driverRightStick = gamepad1.right_stick_x;
+
+        driverLeftStick = Range.clip(driverLeftStick, -1, 1);
+        driverRightStick = Range.clip(driverRightStick, -1, 1);
+
+        driverLeftStick = (float)scaleInput(driverLeftStick);
+        driverRightStick =  (float)scaleInput(driverRightStick);
+    }
+
+    @Override
+    public void stop() {
+
+    }
+
+    double scaleInput(double dVal)  {
```

Comment:
What is actually being sent to this function? `dVal` seems like a really generic name.